torch/_tensor.py

```diff
@@ -1094,25 +1098,43 @@ def unique_consecutive(self, return_inverse=False, return_counts=False, dim=None
         )
 
     @_handle_torch_function_and_wrap_type_error_to_not_implemented
-    def __rsub__(self, other):
+    def __rand__(self, other: Union["Tensor", int, bool]) -> "Tensor":
```

Comment:
We're relying on the commutativity of all three bitwise operators in this code. 

Another way to do it without reversing the order of the arguments would be something like this:

```
return other & super()
```

Which actually looks a bit better now to me if it works as it immediately calls the base class and doesn't switch the paramters.

I'm going to let this run percolate to see if there are any failures before trying that one.

Stay tuned!